chips/cc26x2/src/radio/rfc.rs

```diff
@@ -353,29 +354,29 @@ impl RFCore {
             modulation: {
                 let mut mdl = prop::RfcModulation(0);
                 mdl.set_mod_type(0x01);
-                mdl.set_deviation(0x64);
+                mdl.set_deviation(0x14);
                 mdl.set_deviation_step(0x0);
                 mdl
             },
             symbol_rate: {
                 let mut sr = prop::RfcSymbolRate(0);
                 sr.set_prescale(0xF);
-                sr.set_rate_word(0x8000);
+                sr.set_rate_word(0x3333);
                 sr
             },
-            rx_bandwidth: 0x52,
+            rx_bandwidth: 0x4C,
             preamble_conf: {
                 let mut preamble = prop::RfcPreambleConf(0);
-                preamble.set_num_preamble_bytes(0x4);
+                preamble.set_num_preamble_bytes(0x2);
                 preamble.set_pream_mode(0x0);
                 preamble
             },
             format_conf: {
                 let mut format = prop::RfcFormatConf(0);
                 format.set_num_syncword_bits(0x20);
                 format.set_bit_reversal(false);
-                format.set_msb_first(true);
-                format.set_fec_mode(0x0);
+                format.set_msb_first(false);
+                format.set_fec_mode(0x8);
```

Comment:
mode = 0x8 means? Might be worth a definition

Reply:
I am hesitant to define anything that is still fluid as TI releases new configs with each chip revision and rfstudio settings version

Reply:
Do you know what i means tho?